packages/compass-import-export/src/import/list-csv-fields.ts

```diff
@@ -32,6 +32,8 @@ export async function listCSVFields({
     preview: [],
   };
 
+  // TODO: deal with BOM and utf-8 validation here too
```

Comment:
Missed this in my previous PR where I added tests and validation to only three out of four endpoints by accident. Will make a PR just for that separately.